index.js

```diff
@@ -1,23 +1,22 @@
 #!/usr/bin/env node
 
-// SUPER IMPORTANT:
-// Node is extremely annoying and refuses to allow ESM imports in
-// CLI tools, so we have to use require here. Do not change to import.
```

Comment:
read this

Reply:
ESM works after setting `type: "module"` in `package.json`. Yet to add a test hence PR is in draft mode.

Reply:
can you just do a PR converting to ESM without any other changes and I can test that.

Reply:
Sure

Reply:
Just released the ESM version and tested it via `npx` and it totally worked. Tested on Win 10 and Ubuntu. Can't believe I hadn't tried setting `module` the last time I was setting this up. Great work!

Now we can get this PR updated and reviewed